configuration/init/systemd/c8y-log-plugin.service

```diff
@@ -7,6 +7,7 @@ User=root
 ExecStart=/usr/bin/c8y-log-plugin
 Restart=on-failure
 RestartPreventExitStatus=255
+RestartSec=5
```

Comment:
What is the significance of this? Isn't the downtime will be more, when a service tries to restart?